### PR TITLE
RBAC for composing Releases

### PR DESCRIPTION
**Pull request by scoheb**
- Enable the abilty for the Release team to compose Releases

### CODE REVIEW

**Comment by scoheb** [comment]
@johnbieren @Michkov please review

File: components/authentication/managed-workspaces.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```
Comment (johnbieren) [inline components/authentication/managed-workspaces.yaml:1]: This file should be added in `kustomization.yaml` too, no?

Comment (scoheb) [reply components/authentication/managed-workspaces.yaml:1]: yep! good catch

Comment (scoheb) [reply components/authentication/managed-workspaces.yaml:1]: fixed

**Review by johnbieren — APPROVED** [review]

**Comment by Michkov** [comment]
@ralphbean @MatousJobanek @sbose78 Do you think that managed-namespaces should be handled by infra-deployments?

@scoheb Do you have any ticket regarding this work which could help me to understand?

**Comment by MatousJobanek** [comment]
There is an ongoing discussion in a slack thread. 
https://coreos.slack.com/archives/C02K1781K70/p1671111375323959
TBH, first of all, it needs to be clarified:
* what we are trying to do here
* why in this way
* what is missing in the StoneSoup functionality that we cannot follow the standard way of provisioning workspaces
* if there is anything missing, why do we have to hack the process and our cluster to provide functionality which is not supported yet

**Review by sbose78 — APPROVED** [review]
Valid questions by Matous, we should get to them without necessarily blocking this PR. Context would be great :)

Based on the slack thread, it doesn't look like Scott has any other way of accomplishing this, and if this helps us move the needle, so be it.

**Comment by scoheb** [comment]
No longer needed. I will provide a summary shortly.

**Pull request closed without merging**